utilities/cli/container.py

```diff
@@ -64,7 +64,7 @@ class HoloHubContainer:
 
     # SDK and path configuration
     SDK_PATH = os.environ.get("HOLOHUB_SDK_PATH", "/opt/nvidia/holoscan")
-    BASE_SDK_VERSION = os.environ.get("HOLOHUB_BASE_SDK_VERSION", "3.6.0")
+    BASE_SDK_VERSION = os.environ.get("HOLOHUB_BASE_SDK_VERSION", "3.7.0")
```

Comment:
@wyli the naming format of the containers have been changed in this release. This should be updated too: https://github.com/nvidia-holoscan/holohub/blob/aaddca50a5cb96849b70258683a16494d37bb57f/utilities/cli/container.py#L79

Reply:
Agreed, that’s included in https://github.com/nvidia-holoscan/holohub/pull/1149 under review